lib/runtime/ppx_mysql_runtime.mli

```diff
@@ -61,7 +61,7 @@ module type PPX_CONTEXT = sig
     val execute_null 
       : stmt -> string option array -> (stmt_result, [> wrapped_error]) result IO.t
 
-    val fetch
+    val fetch 
```

Comment:
OCaml Format.

Reply:
Oh, this looks like a bug in ocamlformat.

Reply:
Yeap.

Reply:
I'll report this at some point. For now that's okay.